apps/breethe_web/test/breethe_web/controllers/location_controller_test.exs

```diff
@@ -12,6 +12,11 @@ defmodule BreetheWeb.LocationControllerTest do
     test "when filtering by location name" do
       location = insert(:location, measurements: [])
 
+      measured_at =
+        location.last_updated
+        |> Map.put(:microsecond, {0, 0})
+        |> DateTime.to_iso8601()
+
```

Comment:
Do you think we should isolate this in a private function (it's used in multiple tests) or does it make the tests clearer if it is explicitly there? I don't think the way we're handling time is the point of the tests and so abstracting it away might make them clearer?

Reply:
Sure, this could be in a private function so we reduce duplication - that function should be defined in this file though so we don't split things apart and add confusion.